performance-storage-service/pss_project/api/serializers/database/OLTPBenchResultSerializer.py

```diff
@@ -3,9 +3,9 @@
 from pss_project.api.serializers.fields.UnixEpochDatetimeField import UnixEpochDateTimeField
 
 
-
 class OLTPBenchResultSerializer(ModelSerializer):
     class Meta:
         model = OLTPBenchResult
-        fields = ('time','branch','query_mode','build_id','git_commit_id',
-            'benchmark_type','scale_factor','terminals','duration','weights','metrics')
+        fields = ('time', 'git_branch', 'git_commit_id', 'jenkins_job_id', 'db_version', 'environment', 
+                  'benchmark_type', 'query_mode', 'scale_factor', 'terminals', 'client_time', 'weights', 
+                  'metrics', 'incremental_metrics')
```

Comment:
Ah, was the motivation behind changing the REST attributes to change the column names? If that is the case I think you would just need to update the `convert_to_db_json` function in `OLTPBenchRest`

Reply:
> Ah, was the motivation behind changing the REST attributes to change the column names? If that is the case I think you would just need to update the `convert_to_db_json` function in `OLTPBenchRest`

Is this file auto-generated? Should I just leave it as it was and do not modify it?

Reply:
This file is not auto-generated